github-helper.js

```diff
@@ -0,0 +1,66 @@
+exports.GithubHelper = class {
+  constructor({ client, context }) {
+    this.client = client;
+    this.context = context;
+  }
+
+  async assignToProjects({ columnName, newProjects }) {
```

Comment:
similar issue: the argument name does not make clear whether something is an object or just the name of the object. should this be `newProjectNames`?